signature/policy_eval.go

```diff
@@ -14,10 +14,20 @@ import (
 )
 
 // PolicyRequirementError is an explanatory text for rejecting a signature or an image.
-type PolicyRequirementError string
+type PolicyRequirementError struct {
+	message string
+}
+
+// NewPolicyRequirementError is used to create a new PolicyRequirementError
+func NewPolicyRequirementError(message string) *PolicyRequirementError {
```

Comment:
I don’t think external callers need to create their own instances; they can define their own error types.  So, make this private, please, or perhaps drop it completely and let everyone create `PolicyRequirementError{"…"}`.